pkg/impersonation/impersonation.go

```diff
@@ -47,6 +52,8 @@ func New(userInfo user.Info, clusterContext *config.UserContext) (Impersonator,
 	return impersonator, nil
 }
 
+// SetUpImpersonation creates a service account on a cluster with a clusterrole and binding allowing it to impersonate a Rancher user.
```

Comment:
Binding here means "cluster role binding", right? I'd be explicit about that, since Kubernetes allows to link cluster roles with role bindings OR cluster role bindings.

Reply:
done